### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by f-Xiaoxi**
-Styled nav bar and hid links that require sign in 
-Combined sign in and sign up page so that the page doesn't look empty (used modal for 'forgot password?')
-Added footer with flex-wrapper so footer remains at the bottom even if page content is short
-Tried to fix scroll spy but still not going to the correct position
-Nav bar and footer not responsive yet (will do it later)
-Halfway through checklist 
-Sorry the logo looks terrible but my brain isn't working...

### CODE REVIEW

**Comment by barbayjuliette** [comment]
Sign in / create account: put color when hover?
Scroll spy seems to work. 
The logo is good. Maybe change it to yellow in navbar? 
What do you mean navbar and foorter non responsive? For the navbar, in our PRs the bookmarks/trips are responsive, so should be fine. Only About us page to do.

File: app/assets/stylesheets/components/_checklist.scss [hunk 1]
```diff
@@ -0,0 +1,195 @@
+
+.floating-button{
```
Comment (barbayjuliette) [inline app/assets/stylesheets/components/_checklist.scss:2]: For which checklist is that ? I don't see any checklist anywhere ?

**Comment by barbayjuliette** [comment]
When logged out, when I click on some cards to go to the show page, the show page is scrolled all the way to the bottom. I think it might be related to the big banner 'explore' and its scroll.
I can see the show page has an event scroll, and scroll-behavior: smooth. If I deactivate it, the problem doesn't happen anymore. I think it comes to your JS / CSS for the big banner. DO you know what it could be ?

**Comment by f-Xiaoxi** [comment]
> * The logo is good. Maybe change it to yellow in navbar?
image
Changed it to yellow in the nabber. Looks better?

File: app/assets/stylesheets/components/_checklist.scss (hunk 1 above)
Comment (f-Xiaoxi) [reply app/assets/stylesheets/components/_checklist.scss:2]: Sorry this belongs to the checklist branch. Will remove everything that belongs to the checklist from this branch

**Comment by f-Xiaoxi** [comment]
> * Sign in / create account: put color when hover?
Will perhaps try putting in yellow when hover.

> * What do you mean navbar and foorter non responsive? For the navbar, in our PRs the bookmarks/trips are responsive, so should be fine. Only About us page to do.
Sorry, I meant that the nav bar and footer look messy when we make the screen size smaller. Will fix it later on.

> * When I click on the logo, it should also lead me to all trails.
Ok, will add the link.

> * Reset password, when I reset, looks like it doesn't work, the email stays.
Do we need to send an email for this? Or just to show that an email is sent? (maybe the modal closes after 
you click 'reset password'?)

**Comment by lxingern** [comment]
The "Welcome, <user>" and avatar are wrapping - can make it stay on the same line?
image

**Comment by f-Xiaoxi** [comment]
> The "Welcome, " and avatar are wrapping - can make it stay on the same line?
> image

Adjusted the width. Should be okay now

**Comment by lxingern** [comment]
Ehh now the links text are wrapping... But it's ok la we can still use this for the demo today. I will approve first. You're using grid for the navbar is it? I think it might be easier to use flexbox. 

image

**Review by lxingern — DISMISSED** [review]
Ok to merge now but to look at the text wrapping issue later on.

**Comment by f-Xiaoxi** [comment]
image
oh nooo but It looks ok on mine? Maybe because of the screen size.
Will switch to a dropdown menu when the screen is small

**Review by lxingern — APPROVED** [review]
Ok to merge

**Pull request merged**